website/src/components/Loading.tsx

```diff
@@ -0,0 +1,13 @@
+import CircularProgress from '@mui/material/CircularProgress';
+import Box from '@mui/material/Box';
+
+export function Loading() {
```

Comment:
There appear to be two versions of the loading screen, one without styling, and one with. I saw it when loading the page with network throttling, but you can just return the Loading-component in the `render` Method of the `App.tsx` file like this to see the unstyled version:
```
  render() {
    return (
        <Loading />
    );

```
Here how it looks like when loading (after ~5 seconds):
![2024-03-03_14-21](https://github.com/freifunkMUC/wg-access-server/assets/3245637/2c68c2e7-4943-4190-8d64-ffe555bb8780)

This should be consistently styled.

Reply:
you mean with styled version the one with the Navigation bar? and one without?

Reply:
i added a info what is Loading:
[`8885483` (#594)](https://github.com/freifunkMUC/wg-access-server/pull/594/commits/8885483979b76a7b10187d3eb5cfeaefb0071aa6)

because the Loading with the navigation is because of this:

Reply:
Yes, the differences are:
- 'unstyled': No header, font is in the default font (serif)
- 'styled': Header present, font is set in the configured style (Roboto)

Probably in the beginning the `Loading` component is shown as is (i,.e. without outer page structure and thus styling), and after the successful gathering of the App info the page is rerendered and the loading component is kept.

I was able to have it styled all the time (and, incidentally also observing the dark mode) by wrapping it up in parts of the complete page structure like this:
```
    if (!AppState.info) {
      return (
          <Router>
            <ThemeProvider theme={darkLightTheme}>
              <CssBaseline />
              <Navigation />
              <Box component="div" m={2}>
                <Loading />
              </Box>
            </ThemeProvider>
          </Router>
      );
    }
```

Reply:
added with [`370e996` (#594)](https://github.com/freifunkMUC/wg-access-server/pull/594/commits/370e996eeeefdf4912b64920056f7a9ab19236a8)